app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/ThreadController.kt

```diff
@@ -346,7 +348,7 @@ class ThreadController @Inject constructor(
                             } ?: realmListOf()
 
                             remoteMessage.initLocalValues(
-                                isFullyDownloaded = true,
+                                areHeavyDataFetched = true,
                                 isTrashed = localMessage.isTrashed,
                                 messageIds = localMessage.messageIds,
                                 draftLocalUuid = remoteMessage.getDraftLocalUuid(realm),
```

Comment:
Can we spare a realm call if we simply read the localMessage.draftLocalUuid or does it matter here to recompute it?

Reply:
No, because we've got no assurance that the localMessage will have the `draftLocalUuid`.
Since a Draft can have multiple Messages, we need to use the remote version to be sure we don't lose the data.